packages/primitives/dialog/src/dialog.config.ts

```diff
@@ -38,7 +38,7 @@ type RdxBaseDialogConfig<C> = {
 
     mode?: RdxDialogMode;
 
-    backdropClass?: string;
+    backdropClass?: string | string[];
```

Comment:
should we add a check for an empty array?
https://github.com/radix-ng/primitives/blob/main/packages/primitives/dialog/src/dialog.service.ts#L60

Reply:
No, it’s not necessary. The check (config.backdropClass ? ...) is sufficient, as it covers cases where backdropClass is undefined or null. Adding .length only complicates the code without significant benefit unless there’s a specific need to handle empty values separately.